safety/scan/finder/handlers.py

```
@@ -2,8 +2,8 @@
 import os
 from pathlib import Path
 from types import MappingProxyType
-from typing import Dict, List, Optional, Optional, Tuple
-
+from typing import Dict, List, Optional, Set
```

Ruff

`typing.Set` imported but unused

Remove unused import: `typing.Set`

(F401)